SDK/simVis/Entity.cpp

```diff
@@ -160,7 +160,7 @@ bool CoordSurfaceClamping::isValid() const
 void CoordSurfaceClamping::setMapNode(const osgEarth::MapNode* map)
 {
   mapNode_ = map;
-  workingSet_._lru.clear(); // Probably unnecessary
+  workingSet_ = osgEarth::ElevationPool::WorkingSet(); // Probably unnecessary
```

Comment:
This change requires master osgEarth.  Will need CMake checks to avoid compile errors.  Will start to work on that...

Reply:
Since it was a one liner I just did the solution internally, I've pushed out the fix in https://github.com/USNavalResearchLaboratory/simdissdk/commit/5ff8130561dd9b08b28109e4db42fc1cf7e59be5